kpops/components/streams_bootstrap/streams/model.py

```diff
@@ -183,6 +188,16 @@ class StreamsAppAutoScaling(CamelCaseConfigModel, DescConfigModel):
     )
     model_config = ConfigDict(extra="allow")
 
+    @model_validator(mode="after")
+    def validate_model_config(self) -> Self:
```

Comment:
this will cause an error with Pyright on Python 3.10, for now I believe it's best to 
```suggestion
    def validate_model_config(self) -> StreamsAppAutoScaling:
```

and

```python
from __future__ import annotations
```

perhaps add a comment similar to here
https://github.com/bakdata/kpops/blob/108b25cb569a434107ceecc27591575430cef41a/kpops/component_handlers/kubernetes/model.py#L20

Reply:
-.-
```
 error: Argument of type "(self: Self@StreamsAppAutoScaling) -> StreamsAppAutoScaling" cannot be assigned to parameter of type "_AnyModelAfterValidator[_ModelType@model_validator]"
```

Reply:
hm, in that case we have to remove the return type annotation unfortunately

Reply:
@disrupted Fixed!